akka-actor/src/main/resources/reference.conf

```diff
@@ -276,6 +276,10 @@ akka {
 
         # Max number of threads to cap factor-based parallelism number to
         parallelism-max = 64
+
+        # Set to first-in-first-out scheduling mode when you set "async-mode = on" or
+        # last-in-first-out scheduling mode when you set "async-mode = off".
+        async-mode = on
```

Comment:
I know that this comes from FJP, but it still reads awkwardly: “asynchrony” cannot really be switched on and off with this toggle. I’d prefer to be explicit and parse the strings `LIFO` and `FIFO` (explaining that one is a stack and the other a queue should be enough to give people an idea of what this does).